manifests/service/systemd.pp

```diff
@@ -149,6 +149,14 @@
       $user  = $elasticsearch::elasticsearch_user
       $group = $elasticsearch::elasticsearch_group
 
+      if ($new_init_defaults != undef) {
```

Comment:
this variable will never be undef.
Its best to check if the keys actually exist before using them.